### ISSUE TITLE
Error CRX_REQUIRED_PROOF_MISSING while installing extension through drag and drop of crx file in dist

### ISSUE THREAD

**Issue by pietroppeter** [joshgoebel/se_highlightjs#4]
**Describe the bug**
Installing the extension through drag and drop of the `.crx` file found in `dist` folder, fails with error "Invalid package :'CRX_REQUIRED_PROOF_MISSING'".

**To Reproduce**
1. Download `extension.crx` file in [`dist` folder](https://github.com/joshgoebel/se_highlightjs/tree/main/dist)
2. Open chrome://extensions/ in Chrome browser
3. Drag and drop the file in extension tab
4. Error is seen at the top of content

**Expected behavior**
Extension should be installed without error (the above is option 1 for installation in readme)

**Screenshots**
Screenshot of the error (browser language is italian)

![image](https://user-images.githubusercontent.com/4997234/155001359-9665223d-a315-46bf-bc04-8818cc3d882d.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise
 - Browser: Chrome
 - Version: 98.0.4758.102

**Comment by joshgoebel** [comment]
Perhaps this is relevant?

https://support.google.com/chrome/thread/7757612/what-does-crx-required-proof-missing-mean-when-trying-to-install-a-chrome-extension?hl=en

**Comment by pietroppeter** [comment]
did not really find clues there that help me. found this interesting:

https://stackoverflow.com/questions/57425449/chrome-extension-says-crx-required-proof-missing-while-installing

which lead me to this:

https://stackoverflow.com/a/24577660/4178189

from which I gather that (at least on windows) the only option for installing would be to use developer mode and use the unpacked version.

**Comment by joshgoebel** [comment]
Switching to the dev channel probably isn't terrible... you could also just install the plugin from the web store.  That said, so far there is no new release for v3 of Exercism so there is currently no reason to use the plugin at all.  I've been pretty happy with v3 so far and haven't found the need to start patching it - and it's technically also going to be much more difficult now thanks to all the use of React/Hotwire.

Was there a specific reason you were wanting to install?

**Comment by pietroppeter** [comment]
yes, sure it should not be a big deal to use developer mode, I will test it and report back.

> Was there a specific reason you were wanting to install?

lack of highlighting for Nim in Stack Overflow came up again ([here](https://stackoverflow.com/questions/71199715/have-to-count-a-character-occurrence-in-nim-string-type/71200089#comment125868076_71200089)) and I linked this:

https://meta.stackexchange.com/questions/355491/add-syntax-highlighting-support-for-nim

there I found your comment about this extension (did not know it was used in relation to exercism) and wanted to try it out :)

**Comment by joshgoebel** [comment]
OH, I was getting this confused with my Exercism plugin, LOL.... sorry.  I should probably bump HLJS and release a new version of this.
